src/common/models/Projection.ts

```diff
@@ -445,6 +445,17 @@ export class Projection {
     return this.getColumn(dataset);
   }
 
+  getAnomalies(dataset: DatasetId): Column[] {
+    if (!this.annotations?.newCases?.anomalies) {
+      return [];
+    }
+
+    return this.annotations.newCases?.anomalies.map(item => ({
+      x: new Date(item.date).getTime(),
+      y: item.original_observation,
+    }));
+  }
```

Comment:
@mikelehen what would be a good way to extract the anomalies in `Column[]` format? I just hacked this quickly to check how it would work.